packages/components/menu/Menu.mdx

```diff
@@ -1,24 +1,46 @@
 ---
 title: 'Menu'
-type: 'component'
-status: 'stable'
 slug: /components/menu/
 github: 'https://github.com/contentful/forma-36/tree/master/packages/components/menu'
```

Comment:
Could you also replace this link?

there is no Menu component in master branch, that's way this link takes the user to a 404